functions/isTraceIdRepeated.m

```diff
@@ -0,0 +1,18 @@
+function isTraceIdRepeated(CellsTable,time) %use to debug 
+    rows=CellsTable.Time==time;
+    CellsTable=CellsTable(rows,:);
+    rows=rows(rows==1);
+    for i=1:length(rows)-1
+        currentTrace=CellsTable.Trace(i);
+        for j=1:length(rows)
+            if i~=j
+                trace=CellsTable.Trace(j);
+                if strcmp(currentTrace, trace)==1
+                    error = 'Trace id repeated'
+                end
+            end
+
+        end
+    end
+end
+    
```

Comment:
yes and no
yes -> good in principle
no -> slow in practice, just the law of infinitesimally small probabilities (i made that up)